gensim/models/wrappers/fasttext.py

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 #
-# Author: Jayant Jain <jayantjain1992@gmail.com>
```

Comment:
Why you remove this?

Reply:
Wasn't sure about the header. Feel free to take the code and do whatever you want with it.